.ruby-version

```diff
@@ -1 +1 @@
-2.6.0
+2.6.0-dev
```

Comment:
Is there a reason for this change?

Reply:
no it is was issue with my macos with rbenv.  I am using custom docker ruby 2.6.0 with volume binding to work around it now. It should work with ruby 2.6.0.